client/src/components/helpers/AvatarRenderer.js

```diff
@@ -15,13 +15,31 @@ const propTypes = {
 
 function AvatarRenderer({ picture, userName, isDonating, isTopContributor }) {
   const { t } = useTranslation();
+  const [isPictureValid, setIsPictureValid] = useState(true);
   let borderColor = borderColorPicker(isDonating, isTopContributor);
-  let isPlaceHolderImage =
-    /example.com|identicon.org/.test(picture) || picture === defaultUserImage;
+  const loadEvent = () => setIsPictureValid(true);
+  const errorEvent = () => setIsPictureValid(false);
+  let validationImage = document.createElement('img');
+  validationImage.src = picture;
+
+  useEffect(() => {
+    validationImage.onload = loadEvent();
+  }, [picture, validationImage.onload]);
```

Comment:
```suggestion
  }, [picture]);
```
I don't think `validationImage.onload` is changing between renders, also I think that you can only be conditional on props and state, but I'm not 100% sure.